### DISCUSSION TITLE
PDNS can't update existent zone from database

### DISCUSSION THREAD

**Discussion by Teddyhoang** [PowerDNS/pdns#13212] [Q&A]
- [ ] This is not a support question, I have read [about opensource](https://www.powerdns.com/opensource.html) and will send support questions to the IRC channel, [Github Discussions](https://github.com/PowerDNS/pdns/discussions/) or the mailing list.
- [ ] I have read and understood the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/)

 - Program: Authoritative 
 - Issue type: Bug report

### Short description
- I have backend with databases already exist some zone
- I create authoritative server with that backend
- But when i list zone pdns have it show that no zones is load to pdns

### Environment

 - Operating system: Ubuntu 22.04
 - Software version: authoritative 4.8.1
 - Software source: 

### Steps to reproduce

1.  check zone existent in mysql
I have databases already exist some zone
```
$ mysql -updnsadmin -p
use pdns;
select * from domains;
+----+--------------+--------+------------+--------+-----------------+---------+---------+---------+
| id | name         | master | last_check | type   | notified_serial | account | options | catalog |
+----+--------------+--------+------------+--------+-----------------+---------+---------+---------+
|  1 | example.com  |        |       NULL | NATIVE |            NULL |         | NULL    | NULL    |
+----+--------------+--------+------------+--------+-----------------+---------+---------+---------+
1 rows in set (0.00 sec)
```
2. Authoritative backend config file
```
launch+=gmysql
gmysql-dbname=pdns
gmysql-host=x.x.x.x
gmysql-password=xxxx
gmysql-port=3306
gmysql-user=pdnsadmin
```
with x.x.x.x is mysql backend
- Restart pdns.service
```
$ systemctl restart pdns
```
3. Check zone with pdnsutil
```
$ pdnsutil list-all-zones
Sep 06 15:57:31 gmysql Connection successful. Connected to database 'pdns' on 'x.x.x.x'.
Sep 06 15:57:31 gmysql Connection successful. Connected to database 'pdns' on 'x.x.x.x'.
```

### Expected behaviour

Authoritative auto reload zone that existent in backend

### Actual behaviour

Authoritative unable to load zone from backend

**Comment by zeha** [comment]
This is the most useless bug report I've seen in a while

**Comment by Teddyhoang** [comment]
@zeha would you explain this to me. I have reach dead end

**Comment by omoerbeek** [comment]
A zone needs at least a SOA and NS records. Run `pdnsutil check-zone yourzonename`

You do not seem to have read or understood the two questions asked when submitting this. Please re-read and act accordingly.

Our GitHub issue tracker is for bug reports and feature requests. Your
question looks like a support question. I'm moving this to a discussion.
